.github/config.yml

```diff
@@ -0,0 +1,22 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
```

Comment:
does this file has to be named like this exactly? The name is too generic and it doesn't ring a bell that it's about *some* bot